src/app/applications/[id]/page.tsx

```diff
@@ -28,52 +29,49 @@ export default function ApplicationDetailsPage() {
   const sidebarItems = createApplicationSidebarItems(application);
 
   return (
-    <PageContainer className="xl:grid xl:grid-cols-12 xl:grid-rows-[auto-auto] xl:gap-x-20">
-      <div className="xl:col-span-9 xl:row-span-1">
+    <PageContainer className="flex flex-col items-start justify-start lg:flex-row">
```

Comment:
Removed grid based layout to make it align with other pages and consistent sidebar size